app/controllers/users_controller.rb

```diff
@@ -1,5 +1,5 @@
 class UsersController < ApplicationController
-  before_action :load_user, only: %i(show destroy)
+  before_action :load_user, only: %i(show destroy following followers)
```

Comment:
Thay thế only = except đi. quá 4 r mà

Reply:
anh ơi thay bằng except thì sẽ except 5 cái là index, edit, create, update, new nên em nghĩ để only vẫn hơn chứ ạ

Reply:
dùng only an toàn hơn except e ạ.